packages/host/app/controllers/card.ts

```diff
@@ -2,8 +2,15 @@ import Controller from '@ember/controller';
 import { tracked } from '@glimmer/tracking';
 
 export default class CardController extends Controller {
-  queryParams = ['operatorModeState', 'operatorModeEnabled'];
+  queryParams = [
+    'operatorModeState',
+    'operatorModeEnabled',
+    'sid',
```

Comment:
I suggested to add comments about what `sid` and `clientSecret` query params are for and where they are coming from in one of my previous reviews, I still think it would be very useful to people coming to this controller (and also to the route), as it's not immediately clear from their name what they are for.